src/redux/categories/categories.js

```diff
@@ -0,0 +1,21 @@
+const CHECKSTATUS = 'CHECKSTATUS';
+
```

Comment:
- [ ] As I mentioned in the previous comment as  required for the project, kindly make sure the **action type** uses the [ducks pattern](https://github.com/erikras/ducks-modular-redux) to match the expected result and as each each action is composed by 2 different words, I suggest you separate them with an underscore(`_`)  so to be something like this: `const CHECK_STATUS = .../.../...`